com.microsoft.mrtk.core/Interactables/StatefulInteractable.cs

```diff
@@ -135,6 +135,16 @@ public string SpeechRecognitionKeyword
             Tooltip("If true, then the voice command will only respond to voice commands while this Interactable has focus.")]
         public bool VoiceRequiresFocus { get; private set; } = true;
 
+        [SerializeField, Tooltip("If true, then the OnClick event will only get fired while this Interactable is being hovered.")]
+        private bool requiresHover = false;
+
+        /// <inheritdoc />
+        public bool RequiresHover
```

Comment:
nit: can this be

```c#
/// <summary>Doc</summary>
[field: SerializeField, Tooltip("If true, then the OnClick event will only get fired while this Interactable is being hovered.")]
public bool RequiresHover { get; set; } = false;
```

to match the pattern used in the rest of the file? We started converting most of our files over but didn't get to them all. This was one of the first though.

Reply:
+1!